.github/workflows/checkimage.yaml

```diff
@@ -0,0 +1,27 @@
+name: Regular base image update check
+on:
+  schedule:
+    - cron: "5 0 * * *"
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Install skopeo
+        run: sudo apt-get install -y skopeo
+      - name: Check change
+        run: |
+          skopeo inspect "docker://$(grep -Po '(?<=FROM )([^"]+)' Dockerfile)" | grep -Po '(?<="Digest": ")([^"]+)' > .baseimagedigest
```

Comment:
I would recommend to also check RPMs for new updates, as the project's Dockerfile does `dnf update`.
Check https://github.com/RedHatInsights/eventing-integrations/pull/91/files

Reply:
You are right, I removed it for my dev process and forgot to put it back :-)